src/utils/paths.js

```diff
@@ -1,5 +1,6 @@
-// export const DOMAIN = 'http://localhost:7000/';
-export const DOMAIN = 'https://web-chat-labs.herokuapp.com/';
+export const DOMAIN = 'http://localhost:7000/';
+// export const GUEST_CLIENT_DOMAIN = 'https://webchatlabs-guest.netlify.com/';
+// export const DOMAIN = 'https://web-chat-labs.herokuapp.com/';
```

Comment:
Please change the `DOMAIN` value so it has heroku URL and comment the one with localhost out